Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -181,7 +198,7 @@ public static string FindRichestPerson(string[] peopleAndBalances)
 
         }
 
-        private static decimal FindRichestPerson(string[] peopleAndBalances, decimal HighestBalance, StringBuilder RichestPerson, int i, string[] currentPersonData, decimal highestAmountOfPerson)
+        private static decimal FindMorePeopleWithSameAmount(decimal HighestBalance, StringBuilder RichestPerson, string[] currentPersonData, decimal highestAmountOfPerson)
```

Comment:
"More" is redeundant. This function takes too many parameters, it implies it does too many things.

Reply:
oke, Thanks, Could I maybe place the whole code back into the method that calls it ?

Reply:
to be sure we are talking about the same. The FindRichestPerson is the function that takes too much parameters. Or are we talking about both functions. If we talk about both how I can do it better then. Can you give a a example of a good function definitions if both have too many parameters. If only `FindRichestPerson` has too many. I think I can better place it back on the call because I do not see a way to get rid of the parameters.